templates/paragraphs/paragraph--contact-card.html.twig

```diff
@@ -0,0 +1,52 @@
+{%
+  set classes = [
+    'contact-card',
+    view_mode ? 'contact-card---' ~ view_mode|clean_class,
```

Comment:
This is a convention we're trying to avoid. Combining variables straight to create a css class makes finding said classes impossible with search tools.

Also, if a class is not used anywhere in scss, we should not add it. Drupal is verbose enough without us helping it :D

Here is an verbose example: [{% if design == 'align-left' %}](https://github.com/City-of-Helsinki/drupal-hdbt/blob/69993ce389899198edf6972bc79e892a19174d6f/templates/paragraphs/paragraph--banner.html.twig#L3)

It's a bit annoying to write, but it's really specific what's being done and what should be supported with css. Also, the searching thing.